packages/cli/templates/react/src/main.tsx

```diff
@@ -1,4 +1,3 @@
-import React from 'react'
```

Comment:
Why was React imported in the first place?

Reply:
I guess it was there from a template we used. Until a certain version of React, you had to have that import statement even if you're not using `React`. Otherwise the build failed. Since a certain version of React, it's no longer the case. So we can safely remove it.